README.md

```diff
@@ -90,7 +90,7 @@ Input Parameter Values
 
 - ParameterOverrides:
 
-- ProjectImage: `aws/codebuild/nodejs:4.3.2`
+- ProjectImage: `aws/codebuild/nodejs:8.11.0 `
```

Comment:
The changes are just in the README file?

Reply:
Yes.